stable/golang/Dockerfile.1.11.init

```diff
@@ -8,6 +8,8 @@ RUN mkdir -p /go/pkg/dep
 # Prepare function environment
 RUN ln -s /kubeless $GOPATH/src/kubeless 
 ADD /compile-function.sh /
+WORKDIR /	
+RUN chmod +x compile-function.sh	
```

Comment:
you don't need the WORKDIR command (just do chmod +x /compile-function.sh) even though it's weird that you needed to do that.

Reply:
Without the permission, the pod fails to run with `Init:CrashLoopBackOff`, and when I check the logs of `compile` init container, it shows `sh: 1: /compile-function.sh: Permission denied` 

Now if I am not setting the working directory,  I get this error: `chmod: cannot access 'compile-function.sh': No such file or directory`, so I added that too

Reply:
I mean that you can use just one line if you use `/compile-function.sh` instead than `compile-function.sh`

Reply:
right, fixing this.